index.html

```diff
@@ -0,0 +1,1080 @@
+                            <li style="margin-right: 10px;border-right: 1px solid rgb(233,235,237);padding-right: 10px;font-size: 12px;margin-bottom: 5px;">Pakistan</li>
+                            <li style="margin-right: 10px;border-right: 1px solid rgb(233,235,237);padding-right: 10px;font-size: 12px;margin-bottom: 5px;">Polska</li>
+                            <li style="margin-right: 10px;border-right: 1px solid rgb(233,235,237);padding-right: 10px;font-size: 12px;margin-bottom: 5px;"><a href="https://newsweek.ro/">România</a></li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col flex-wrap">
+                        <ul class="d-flex flex-wrap">
+                            <li style="font-size: 12px;margin-right: 10px;padding-right: 10px;border-right: 1px solid rgb(237,237,237);margin-bottom: 5px;">About Us</li>
+                            <li style="font-size: 12px;margin-right: 10px;padding-right: 10px;border-right: 1px solid rgb(237,237,237);margin-bottom: 5px;">Corrections</li>
+                            <li style="font-size: 12px;margin-right: 10px;padding-right: 10px;border-right: 1px solid rgb(237,237,237);margin-bottom: 5px;">Contact Us</li>
+                            <li style="font-size: 12px;margin-right: 10px;padding-right: 10px;border-right: 1px solid rgb(237,237,237);margin-bottom: 5px;">Editorial Guidelines</li>
+                            <li style="font-size: 12px;margin-right: 10px;padding-right: 10px;border-right: 1px solid rgb(237,237,237);margin-bottom: 5px;">Advertise</li>
+                            <li style="font-size: 12px;margin-right: 10px;padding-right: 10px;border-right: 1px solid rgb(237,237,237);margin-bottom: 5px;">Copyright</li>
+                            <li style="font-size: 12px;margin-right: 10px;padding-right: 10px;border-right: 1px solid rgb(237,237,237);margin-bottom: 5px;">Terms &amp; Conditions</li>
+                            <li style="font-size: 12px;margin-right: 10px;padding-right: 10px;border-right: 1px solid rgb(237,237,237);margin-bottom: 5px;">Privacy Policy</li>
+                            <li style="font-size: 12px;margin-right: 10px;padding-right: 10px;border-right: 1px solid rgb(237,237,237);margin-bottom: 5px;">Cookie Policy</li>
+                            <li style="font-size: 12px;margin-right: 10px;padding-right: 10px;border-right: 1px solid rgb(237,237,237);margin-bottom: 5px;">Terms of Sale</li>
+                            <li style="font-size: 12px;margin-right: 10px;padding-right: 10px;border-right: 1px solid rgb(237,237,237);margin-bottom: 5px;">Archive</li>
+                            <li style="font-size: 12px;margin-right: 10px;padding-right: 10px;border-right: 1px solid rgb(237,237,237);margin-bottom: 5px;">Announcements</li>
+                        </ul>
```

Comment:
- [ ] these elements, should be links as the original website. the `<ul>` + `<li>` with `<a>` links inside each menu item.